terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars

```diff
@@ -0,0 +1,30 @@
+environment = "staging"
+aws_profile = "staging-eu-west-2"
+
+# scaling configs
+desired_task_count = 2 # use multi instance in staging
+
+log_level = "debug"
+chs_url = "https://staging.company-information.service.gov.uk"
+piwik_url = "https://matomo.platform.aws.chdev.org/"
+piwik_site_id = "15"
+piwik_start_goal_id = "23"
+piwik_update_start_goal_id = "26"
+cdn_host = "d6nh3dxv55e16.cloudfront.net"
+cookie_domain = "company-information.service.gov.uk"
+account_local_url = "https://identity-staging.company-information.service.gov.uk"
+show_service_offline_page = "false"
+feature_flag_enable_update_statement_validation_05072023 = "false"
```

Comment:
feature_flag_enable_update_manage_trusts_29082023 = "false"
feature_flag_disable_update_private_data_fetch_2806202 = "false"

Reply:
were are adding feature_flag_enable_update_statement_validation_05072023  on line 17 so assuming we just add all.

Reply:
These are the values that are currently in live. Any new values will be added in a separate PR.